app/static/src/plot.ts

```diff
@@ -81,6 +81,7 @@ export function discreteSeriesSetToSkadiChart(
   const series = showIndividualTraces ? s.values : s.values.filter((el) => el.description !== "Individual");
   return series.map((values: OdinSeriesSetValues) => {
     const isIndividual = values.description === "Individual";
+    const name = values.description === "Mean" ? values.name + " (mean)" : values.name;
```

Comment:
This is unrelated opportunistic fix to implement the mean labels which were in the plotly version?